static/js/pages/candidate-single.js

```diff
@@ -70,7 +70,7 @@ function initFilingsTable() {
     path: path,
     columns: filingsColumns,
     order: [[2, 'desc']],
-    dom: tables.simpleDOM,
+    dom: 't<"results-info"lpi>',
```

Comment:
I wasn't sure if `simpleDOM` was used elsewhere yet. But if those are the only places it's being used then I'll replace it. Thanks!